homeassistant/components/motion_blinds/cover.py

```diff
@@ -146,6 +169,37 @@ def is_closed(self):
         """Return if the cover is closed or not."""
         return self._blind.position == 100
 
+    @callback
+    def push_callback(self):
+        """Update entity state when a push has been received."""
+        self.schedule_update_ha_state(force_refresh=False)
+
+    async def async_added_to_hass(self):
+        """Subscribe to multicast pushes and register signal handler."""
+        self._blind.Register_callback(self.unique_id, self.push_callback)
+        async_dispatcher_connect(self.hass, DOMAIN, self.signal_handler)
```

Comment:
I think a best practice is to unregister this when the entity is removed
```suggestion
        self.async_on_remove(async_dispatcher_connect(self.hass, DOMAIN, self.signal_handler))
```

Reply:
agreed